src/python/pants/backend/python/tasks/unpack_wheels.py

```diff
@@ -49,20 +48,37 @@ def subsystem_dependencies(cls):
             PythonSetup,
         )
 
-    class _NativeCodeExtractionSetupFailure(Exception):
+    class SingleDistExtractionError(Exception):
         pass
 
     def _get_matching_wheel(self, pex_path, interpreter, requirements, module_name):
-        """Use PexBuilderWrapper to resolve a single wheel from the requirement specs using pex."""
+        """Use PexBuilderWrapper to resolve a single wheel from the requirement specs using pex.
+
+        N.B.: The resolved wheel is already "unpacked" by PEX. More accurately, it's installed in a
+        chroot.
```

Comment:
Where does this chroot live? In the Pex cache, correct? Probably worth mentioning given how many different chroots Pants deals with.

Reply:
Yes, but I don't understand how that is relevant. The installed ("unpacked"+) wheel is immediately copied in to the v1 task vt.results_dir so the source is of no long term concern to the task, ie it's not symlinked, etc.

We use a private location for the pex cache, ie: it's not ~/.pex but PythonSetup.resolver_cache_dir, so its stored somewhere under there down under some intermediate dirs that are sha1 hashes. Basically, a lot of explaining if the point is to help someone debug this in a non-misleading way.